.github/workflows/ci.yml

```diff
@@ -127,7 +127,30 @@ jobs:
       # Upload the various reports to sonar
       - name: Upload report to SonarCloud
         env:
-          GITHUB_TOKEN: ${{ secrets.TOKEN_GITHUB }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Question: why was this changed ?

Reply:
I initially created a TOKEN_GITHUB secret with a handmade token, but GitHub, when launching the CI, gives a GITHUB_TOKEN by default, which is a secret specifically created for this specific CI run. This is just a security improvement